Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardInputView.swift

```diff
@@ -121,6 +121,7 @@ struct CreditCardInputView: View {
     }
 
     func rightBarButton() -> some View {
+        let disableButtonColor = Color(themeVal.theme.colors.textSecondary)
```

Comment:
Hi,

Let's add this on top as a state variable
https://github.com/mozilla-mobile/firefox-ios/pull/15135/files#diff-0c9c1ea32b51fa6c1159fff83397db9c423a55aa9494e4196e0c1119a2f09f44R20

and use our `applyTheme` method to aggregate all colors for the button
https://github.com/mozilla-mobile/firefox-ios/pull/15135/files#diff-0c9c1ea32b51fa6c1159fff83397db9c423a55aa9494e4196e0c1119a2f09f44R116

Reply:
Ok perfect I'm going to do it.